### DISCUSSION TITLE
QEditor - Option to remove formatting of pasted text

### DISCUSSION THREAD

**Discussion by cimpok** [quasarframework/quasar#17248] [Ideas / Proposals]
It would be great to have a behaviour flag for the QEditor that results removing any formatting from the pasted text when the user inserts text with CTRL-V from the clipboard. This new flag could be called  'plaintext-paste'
The user often copies text from an existing document (or from multiple locations) into a QEdit component which most often results in unwanted formatting.

**Comment by hazzik** [comment]
There is an example in the documentation how to do so: https://quasar.dev/vue-components/editor#plaintext-pasting

**Comment by cimpok** [comment]
Thank you @hazzik, I knew about that workaround, that's why I'm suggesting/asking if this could perhaps be built in the framework.   
I use QEditors a lot and keep repeating adding this event handler method to almost all my components.     .but this code is half-a-page long and makes my code look clumsy.   Isn't there a cleaner solution to this or at least let's hide it into the framework. 
 ...according to my use cases and preference this should be the standard behaviour of QEditors.
